zebra/if_netlink.c

```diff
@@ -1454,6 +1453,326 @@ int netlink_interface_addr(struct nlmsghdr *h, ns_id_t ns_id, int startup)
 	return 0;
 }
 
+/*
+ * Parse and validate an incoming interface address change message,
+ * generating a dplane context object.
+ * This runs in the dplane pthread; the context is enqueued to the
+ * main pthread for processing.
+ */
+int netlink_interface_addr_dplane(struct nlmsghdr *h, ns_id_t ns_id,
+				  int startup /*ignored*/)
```

Comment:
The argument is not used in this path, but it needs to be present because there's a series of apis in the netlink read/process path that require certain function signatures.

Reply:
Ok. Got it.